From 371cc3bed731572b086561b37d7e01d663ac3a2e Mon Sep 17 00:00:00 2001
From: "Baldur" <baldurt@email.com>
Date: Wed, 15 May 2024 08:09:42 +0200
Subject: [PATCH] In fedora coreos 40 the /usr/libexec path is readonly. crio
 tries to create for all his plugin dirs a mkdir and therfore fails on
 /usr/libexec/cni. This causes crio not to start and in consequence kubelet
 fails to start

---
 roles/container-engine/cri-o/templates/crio.conf.j2 | 1 -
 1 file changed, 1 deletion(-)

diff --git a/roles/container-engine/cri-o/templates/crio.conf.j2 b/roles/container-engine/cri-o/templates/crio.conf.j2
index ee673f119..4a56d0621 100644
--- a/roles/container-engine/cri-o/templates/crio.conf.j2
+++ b/roles/container-engine/cri-o/templates/crio.conf.j2
@@ -370,7 +370,6 @@ network_dir = "/etc/cni/net.d/"
 # Paths to directories where CNI plugin binaries are located.
 plugin_dirs = [
 	"/opt/cni/bin",
-	"/usr/libexec/cni",
 ]
 
 # A necessary configuration for Prometheus based metrics retrieval
-- 
2.44.0

